src/utils/daos_hdlr.c

```diff
@@ -205,6 +207,111 @@ cont_create_uns_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
```

Comment:
You should add some comment to describe the purpose of uns_insert action here, and also in daos command on-line help